crates/ruff_linter/src/rules/pyupgrade/rules/redundant_open_modes.rs

```diff
@@ -40,18 +39,18 @@ impl AlwaysFixableViolation for RedundantOpenModes {
     fn message(&self) -> String {
         let RedundantOpenModes { replacement } = self;
         if replacement.is_empty() {
-            "Unnecessary open mode parameters".to_string()
+            "Unnecessary open mode argument".to_string()
         } else {
-            format!("Unnecessary open mode parameters, use \"{replacement}\"")
+            format!("Unnecessary open modes, use `{replacement}`")
```

Comment:
```suggestion
            format!("Unnecessary open mode argument, use `{replacement}`")
```

Should we use the singular form here too or are there cases where there are multiple possible open modes?

Reply:
That's fair. I still think singular makes sense here because even with `rb`, there's only a single `mode` argument (the mode might be multiple characters but it's always a single argument)

Reply:
I'm then leaning towards removing the `argument` everywhere and instead just use `Unnecessary open mode, use {replacement}`

Reply:
That's what I had in mind, based on your explanation but please let me know if you've concerns. I'm sure you know the rule much better by now than I do